manifests/check_vault_daemonset.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: test-vault-connection
+  namespace: default
+spec:
+  selector:
+    matchLabels:
+      name: test-vault-connection
+  template:
+    metadata:
+      labels:
+        name: test-vault-connection
+    spec:
+      tolerations:
+        - key: "nvidia.com/gpu.product"
+          operator: "Exists"
+          effect: "NoSchedule"
+      containers:
+      - name: alpine
+        image: alpine/curl
```

Please use the test image here.

```suggestion
        image: ghcr.io/ocp-on-nerc/openshift-tests:latest
```

---

If we need to add `curl` to the test image, just update the `Containerfile`.